meerk40t/device/gui/defaultactions.py

```diff
@@ -4,16 +4,16 @@
     STD_ICON_SIZE,
     icons8_bell_20,
     icons8_close_window_20,
-    icons8_down_50,
-    icons8_home_20,
+    icons8_down,
+    icons8_home_filled,
     icons8_input_20,
     icons8_output_20,
     icons8_remove_25,
     icons8_return_20,
     icons8_stop_gesture_20,
```

Comment:
These have size too.

Reply:
And rightly so, as they are still bitmapped icons